swap_meet/clothing.py

```diff
@@ -1,2 +1,13 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    """Inherits from 'Item' class and creates a 'Clothing' object."""
+    
+    def __init__(self, category= "Clothing", **kwargs):
```

Comment:
Super cool that you're using `**kwargs` here! This is a handy power that Python functions have but I would actually recommend against using them here for a couple reason. 
1. `**kwargs` allow users to provide any number of arguments with any names they would like. This is dangerous because it leaves room for users to misuse the function.
2. It's less clear how to use the function. If I were a developer trying to use this class, it would be very unclear what arguments I should provide.

For these reasons, it's best to be very clear about how many parameters are expected and what they are. `**kwargs` work best when your function is expecting varying amounts of input. Love that you're exploring new concepts though! 🙌🏾